statistics/handle/bootstrap.go

```diff
@@ -310,24 +444,35 @@ func (h *Handle) InitStats(is infoschema.InfoSchema) (err error) {
 	if err != nil {
 		return err
 	}
-	cache, err := h.initStatsMeta(is)
+	tables, err := h.initStatsMeta(is)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	err = h.initStatsHistograms(is, tables)
 	if err != nil {
 		return errors.Trace(err)
 	}
-	err = h.initStatsHistograms(is, &cache)
+	err = h.initCMSketch4Indices(is, tables)
```

Comment:
why separating the stats cache loading logic for Histogram and CM-Sketch?

Reply:
in previous implementation, initStatsHistograms load all the histogram metadata and cmsketch, because cmsketch statisticdata consume many memusage, so we want to limit the size of cmsketch.
in conclusion,we don't limit initStatsHistograms chunk size , but  initCMSketch4Indices should limit chunk size.